test/Microsoft.ML.Benchmarks/BigramAndTrigramBenchMark.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Benchmarks
+{
+    // Adding this class to not print anything to the console.
+    // This is required for the current version of BenchmarkDotNet
+    internal class EmptyWriter : TextWriter
+    {
+        internal static readonly EmptyWriter Instance = new EmptyWriter();
+        public override Encoding Encoding => null;
+    }
+
+    public class BigramAndTrigramBenchmark
+    {
+        private string s_dataPath_Wiki;
+        private string s_modelPath_Wiki;
```

Comment:
Since these are instance fields, they shouldn't be prefixed with `s_`.  They should be just `_`